.github/workflows/linting.yml

```diff
@@ -4,37 +4,44 @@ on: [push]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
         node-version: [12.x]
 
     steps:
-    - uses: actions/checkout@v1
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: ${{ matrix.node-version }}
-    - name: npm install and build malte-common
-      working-directory: packages/malte-common
-      run: |
-        npm ci
-        npm run build
-      env:
-        CI: true
-    - name: npm install and check styling errors in frontend
-      working-directory: packages/frontend
-      run: |
-        npm ci
-        npm run lint
-    - name: npm install and check styling errors in backend
-      working-directory: packages/backend
-      run: |
-        npm ci
-        npm run lint
-      env:
-        CI: true
-
-
+      - uses: actions/checkout@v1
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: npm install and build malte-common
+        working-directory: packages/malte-common
+        run: |
+          npm ci
+          npm run build
+        env:
+          CI: true
+      - name: npm install and check styling errors in frontend
+        working-directory: packages/frontend
+        run: |
+          npm ci
+          npm run lint
+      - name: npm install and check styling errors in backend
+        working-directory: packages/backend
+        run: |
+          npm ci
+          npm run lint
+      - name: npm install and check styling errors in rga
+        working-directory: packages/rga
+        run: |
+          npm ci
+          npm run lint
+      - name: npm install and check styling errors in malte-common
+        working-directory: packages/malte-common
+        run: |
+          npm ci
```

Comment:
Don't have to run npm ci here since we do it in the first step `npm install and build malte-common` anyway
```suggestion
```